src/backend/api/users/user.controller.js

```diff
@@ -0,0 +1,23 @@
+const { HttpException } = require('../../common/httpException/HttpException');
+const { InValidHttpResponse } = require('../../common/response/invalidHttp.response');
+const { ValidHttpResponse } = require('../../common/response/validHttp.response');
+const { UserService } = require('../../modules/users/user.service');
+const User = require('../../model/user.model');
+
+class Controller {
+    createUser = async (req, res) => {
```

Comment:
You should name it createOne. When calling *UserController.createUser()*, the word user is duplicate and redundant